src/transformers/trainer.py

```diff
@@ -461,10 +461,6 @@ def __init__(
             ):
                 self.backward_prefetch = BackwardPrefetch.BACKWARD_POST
 
-            self.forward_prefetch = False
-            if self.args.fsdp_config.get("forward_prefect", False):
-                self.forward_prefetch = True
```

Comment:
Why are we removing this? Is it done somewhere else?

Reply:
Hello, this is present at this line: https://github.com/huggingface/transformers/pull/25107/files#diff-bfceaff300c851b8e24fc50dc6638482abaec8f7d2a718e877c3828c166bcf79R1614